bundles/org.openhab.binding.touchwand/src/main/java/org/openhab/binding/touchwand/internal/discovery/TouchWandUnitDiscoveryService.java

```diff
@@ -96,7 +95,7 @@ protected void startScan() {
                         }
                         String type = touchWandUnit.getType();
                         if (!Arrays.asList(SUPPORTED_TOUCHWAND_TYPES).contains(type)) {
-                            logger.debug("Unit discovery skipping unsupported unit type : {} ", type);
+                            logger.info("Unit discovery skipping unsupported unit type : {} ", type);
```

Comment:
Please keep debug here.

```suggestion
                            logger.debug("Unit discovery skipping unsupported unit type : {} ", type);
```